package.json

```diff
@@ -67,10 +67,12 @@
     "eslint-plugin-prettier": "^5.2.1",
     "glob": "7.1.7",
     "import-meta-resolve": "^2.2.2",
+    "mermaid": "^11.3.0",
```

Comment:
we also want to probe agoric-sdk CI against this branch. This is to avoid any underlying package used by these new packages to break it (it's broken right now so we may have to wait for https://github.com/Agoric/documentation/pull/1240 to merge and then rebase)